### ISSUE TITLE
new 5.2 series releases are missing commits from the 5-2-stable branch

### ISSUE THREAD

**Issue by stevecrozz** [rails/rails#39371]
I'm prevented from upgrading to the latest in the 5.2 series because I'm still using Ruby 2.2 (I know). But the fixes to restore Ruby 2.2 compatibility do live in the 5-2-stable branch. In fact, there are many commits from the stable branch that do not appear in v5.2.4.3. What is the explanation?

~~~
git cherry -v v5.2.4.3 5-2-stable
+ 0b38f4ceab0c42d0762882152056e5e54b76f3f8 Merge pull request #37849 from kamipo/fix_since_and_ago
+ ea3b42f26ea0c57d30c2f69dd2fdef3b4bfce869 Merge pull request #37862 from giraffate/fix_random_ci_failure
+ b2bf70554c5ba5d9ea12fc368d7a05b82dcbf39c Merge pull request #33217 from eugeneius/test_duration_preserve_parts
+ 39a8266a4c061d447b679f755dd1ed8485a9397f Merge pull request #37882 from kamipo/should_keep_zero_duration_parts
+ 892eab777c418135ce0646e91bc9ebb08a29ab9b Fix syntax error with Ruby 2.2
+ ee1278cc841a883835918fbedb70add96c84543b Rails 5.2 has to support Ruby 2.2 that cannot parse safe navigation operator
+ 13a07268160c1093d4b0ff40e45e487d635fa6b9 Fix ActionController::TestSession#id to return Rack::Session::SessionId instance (#38063)
+ c8c015e40170334ce78a808a3f7008d23bda2fa1 Merge pull request #38128 from kddeisz/forward-kwargs
+ 8868cae884a843c7814148c81cae4c99ba798f7b Rails 5.2 doesn't have `options` method
+ 19138b96cee34e6616c662346cd0728221cc4b0b Merge pull request #38216 from kamipo/fix_failure_for_rack-2.1.0
+ ae3f05e933ec4c42e215c853f17b5bf57e2ed451 Fix `test_vanilla_cookie_with_expires_set_relatively` failure
+ 96097dd740cebbae15e54c18426bb6f0f5686ed4 Fix duration test failure on Ruby 2.4
+ 6c9800954e7137b442f962aed54496fc023bf35b Fix the regex to make `(\.0)` optional
+ 32cfb82922bfc905b14d7d025287dac0b85b1668 Merge pull request #37262 from nobu/Update-for-Time#inspect
+ ddf1cb5ec93dda927a555a57245111d9ba2395e4 Merge pull request #38731 from robin850/fix-sdoc-version
+ 299fe07b4e9585f6405a005c7beefef64bc32ca9 Add CHANGELOG entry to 5.2.4.2
+ a8844200031e40b9a452bc9374de1f7b9440ac7f Ruby 2.2 doesn't support Safe Navigation Operator
+ 08edf165ae52aa22a8ba90de86729e1a4423459c Fix "Exception: ActionView::Template::Error: wrong number of arguments (2 for 1)"
+ f73d439435f9130b34f1595ffe2fdc627b174f43 Merge pull request #38422 from vipulnsward/fix-wrong-spec
+ 97d3432273c41831497c65de3650800b1fb09495 Skip cookie comma separation test
+ abd27d5121573ab591d1501afa5f81e2e9da8c90 Fix `test/controller/request_forgery_protection_test.rb` failure
+ 47b3bc160e8e6045462df0617da55c3d95c8cd69 Fix `test_two_classes_autoloading` failure
~~~

**Comment by rafaelfranca** [comment]
The security releases are not made from the stable branches, but from the current release tag. As 5.2 is now only supported for security releases, I doubt we would have a bug fix release, but maybe we will do a 5.2.5 including all commits from the 5-2-stable branch.

**Comment by stevecrozz** [comment]
@rafaelfranca If 5.2 still supports security releases for Ruby 2.2, then release v5.2.4.3 is inadequate since it doesn't work on Ruby 2.2. Do you disagree?

**Comment by rafaelfranca** [comment]
I agree a little bit. Ruby 2.2 is not supported by the Ruby team anymore since 2018, even for security releases, so in that sense we doesn't consider that release inadequate. But that doesn't means we should drop support to Ruby 2.2 in Rails 5.2, because of that the fix for the Ruby 2.2 support is already in the 5-2-stable branch so people who need to use that version of Ruby can still point to that branch until we have a reason to release that branch.
